src/components/JobSummary/JobSummary.js

```diff
@@ -11,7 +11,7 @@ import { Summary } from './Summary/Summary.js'
 const Header = ({ jobDetails, task, showFileDetails }) => (
     <div className={styles.header}>
         <h3 className={styles.title}>{`${i18n.t('Job summary')}`}</h3>
-        {showFileDetails && (
+        {showFileDetails && jobDetails?.files?.length && (
```

Comment:
@ismay could you have a quick look here?

If you think this could be a good generic fix, then it'd make sense to keep the current code. Otherwise, I think my suggestion makes more sense.

Reply:
Originally I tried to set `showFileDetails` to false in EarthEngineImport (there are no files to show). But if I navigated to a different importer, then back to EarthEngineImport, then the JobSummary re-rendered with showFileDetails = true, which crashed the importer. Not sure why `true` was being used. This was a quick and dirty solution bc I didn't have time to track down the showFileDetails issue.

Reply:
Cool. Makes sense to me too. I'll resolve this conversation.